ngraph_bridge/enable_variable_ops/ops/ngraph_ops.cc

```diff
@@ -120,5 +120,25 @@ REGISTER_OP("NGraphVariableUpdateNGTensor")
         "nGraph variable update NG tensor op. For updating the NG Tensor when "
         "TF tensor is modified by a TF variable modifier op");
 
+// // ------------------------------------------------------------------
```

Comment:
The registration is commented

Reply:
Thanks, I was trying the test again and forgot to uncomment it